src/Redux/books/book.js

```diff
@@ -0,0 +1,32 @@
+import { data } from '../../Components/data';
+/* eslint-disable no-case-declarations */
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const initialState = data;
```

Comment:
- [ ] According to the project description `initialState` should be initialized to an empty array. Kindly do this.

Reply:
Good evening @TedLivist , thank you for your review. But please just to confirm, I put in data because I didn't want the page to be empty when the page loads. The goal of the project is to add books and remove books from the bookstore. which I think I have fully implemented.

Reply:
I know that. I understand. However, this is a requirement listed in the guidelines. Your add and delete functionalities work too with an initial empty page anyways.

Reply:
@mckent05 I know that. I understand. However, this is a requirement listed in the guidelines. Your add and delete functionalities work too with an initial empty page anyways.